src/main/java/net/snowflake/hivemetastoreconnector/core/SnowflakeClient.java

```diff
@@ -84,8 +87,20 @@ public static void generateAndExecuteSnowflakeStatements(
     List<String> commandList;
     try
     {
+      log.info("Checking if " + command.getDatabaseName() + " is in the configured schema list.");
+      Set<String> schemaSet = new HashSet<>(snowflakeConf.getStringCollection(
```

Comment:
seems like we have this logic repeated a couple places. Can we put this in a helper method?

Reply:
Moved the logic to a util class.